src/components/Map.tsx

```diff
@@ -42,8 +48,22 @@ function Map() {
         expanded: true
       });
 
+      const layerList = new LayerList({
+        view: view
```

Comment:
Nitpick: `view: view` can be shortened to `view` like it is on line 56

Reply:
Updated!